lib/views/name.erb

```diff
@@ -0,0 +1,6 @@
+<% user = User.create(:name => 'Casper')
+   user.save %>
```

Comment:
Don't put this kind of logic in the view. You should do things like this in the controller, then store them in instance variables (`@name`, for instance)